templates/flutter/.github/workflows/publish.yml.twig

```diff
@@ -0,0 +1,14 @@
+name: Publish to pub.dev
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+.*'
```

Comment:
We need to escape the periods here to match literal otherwise they'll match any character:

```suggestion
      - '[0-9]+\.[0-9]+\.[0-9]+.*'
```